tests/files/odl/feedbooks_bibliographic.atom

```diff
@@ -254,46 +254,54 @@
   <category term="florida everglades" label="florida everglades"/>
   <category term="united states department of the interior" label="united states department of the interior"/>
 </entry>
-<entry>
```

Comment:
I replaced this entire OPDS entry (Feedbooks-protected audiobook) with one from a real feed.